tools/ansible-runner/Dockerfile

```diff
@@ -19,4 +19,8 @@ RUN echo "[local]" >> /etc/ansible/hosts && \
 
 ADD ./tests ./
 
+RUN mkdir -p ./openebs
```

Comment:
Let us not create a new dirs in the ansible-runner image. Instead, the job has to make an appropriate reference to the path before running the desired playbooks.